pyproject.toml

```diff
@@ -34,6 +34,8 @@ pylint = "^2.17"
 pytest-cov = "^3.0.0"
 pytest-env = "^0.8.1"
 pytest-benchmark = { version = "^4.0.0", extras = ["histogram"] }
+qibojit = "^0.1.3"
```

Comment:
Why is qibojit here again?

Reply:
Thanks. Removed both `qibojit` and `numba`.